<launch>

    <!-- Arguments -->
    <arg name="asl_folder" value="/home/bjoshi/bbdos_shipwreck"/>
    <arg name="scale" value="1.0"/>
    <arg name="compressed" value="true"/>
    <arg name="bag_filename" value="/home/bjoshi/Downloads/afrl_aqua_bbdos_shipwreck.bag"/>

    <arg name="left" value="/cam_fl/image_raw"/>
    <arg name="right" value="/cam_fr/image_raw"/>
    <arg name="imu" value="/imu/imu"/>

    <node name="gopro" pkg="utils_ros" type="extract_bag" output="screen" required="true">

        <param name="asl_folder" value="$(arg asl_folder)"/>
        <param name="bag_filename" value="$(arg bag_filename)"/>

        <param name="left_image_topic" value="$(arg left)"/>
        <param name="right_image_topic" value="$(arg right)"/>
        <param name="imu_topic" value="$(arg imu)"/>
        <param name="compressed" value="$(arg compressed)"/>
        <param name="scale" value="$(arg scale)"/>

    </node>

</launch>


